OSX build is failing

After #90 the build is failing on OSX

```
[ 69%] Building C object src/natwm/CMakeFiles/natwm.dir/natwm.c.o
/Users/<user>/personal/c/natwm/src/natwm/natwm.c:114:9: error: implicit
      declaration of function 'sigemptyset' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
        sigemptyset(&action.sa_mask);
        ^
/Users/<user>/personal/c/natwm/src/natwm/natwm.c:116:9: error: implicit
      declaration of function 'sigaddset' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
        sigaddset(&action.sa_mask, SIGTERM);
        ^
/Users/<user>/personal/c/natwm/src/natwm/natwm.c:123:13: error: implicit
      declaration of function 'sigaction' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
        if (sigaction(SIGTERM, &action, NULL) == -1
            ^
3 errors generated.
```

--- a/src/natwm/natwm.c
+++ b/src/natwm/natwm.c
@@ -4,10 +4,10 @@
 
 #include <getopt.h>
 #include <pthread.h>
+#include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/select.h>
-#include <sys/signal.h>
 #include <unistd.h>
 #include <xcb/xcb.h>
 #include <xcb/xcb_util.h>

